### DISCUSSION TITLE
multi-module project with gradle

### DISCUSSION THREAD

**Discussion by uPagge** [quarkusio/quarkus#28802] [Q&A]
Hello
I'm trying to build a multi-module project with gradle, but I don't have much experience with gradle and something is failing.

<img width="1672" alt="Снимок экрана 2022-10-25 в 03 42 30" src="https://user-images.githubusercontent.com/40397740/197656182-f42473f7-2d65-4b6a-9f29-ec3fa5e04f72.png">

Any ideas what it could be related to?

[a comment by an automated account is not shown]

**Comment by glefloch** [comment]
Hi, could you share a reproducer ? How did you create the project?

**Reply by uPagge** [reply]
I figured out the conflict was with the Dependency Management Plugin. Removed it and was able to start everything. The only thing is that you have to use index-dependency to explicitly specify the dependencies. For some reason Jandex is not working

**Reply by codespearhead** [reply]
@uPagge Can you [close this discussion then](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion)?
